.env

```diff
@@ -1,4 +1,4 @@
 PORT=4000
-MONGO_URI="mongodb://localhost:27017"
+MONGO_URI=""
 ACCESS_TOKEN_SECRET=''
-REFRESH_TOKEN_SECRET='
\ No newline at end of file
+ 
```

Comment:
Let's move these two to `.env.local.template` that's where we need to store these template values